library_website/settings/base.py

```diff
@@ -612,3 +612,4 @@
 # we wish to keep out of sync. People listed here will not show
 # up in the "out of sync staff members" email report.
 DO_NOT_SYNC = []
+MAIL_ALIASES_PATH = '/data/web/aliases/aliases.json'
```

Comment:
I like how this config doesn't require us to maintain anything in the `local.py` and the code fails gracefully when the value is "wrong" (in local dev) :+1: